app/javascript/dashboard/routes/dashboard/notifications/components/NotificationPanel.vue

```diff
@@ -199,6 +207,21 @@ export default {
     onMarkAllDoneClick() {
       this.$store.dispatch('notifications/readAll');
     },
+    openAudioNotificationSettings() {
+      this.$router.push({ name: 'profile_settings_index' });
+      this.closeNotificationPanel();
+      this.$nextTick(() => {
+        const audioSettings = document.getElementsByClassName(
+          'audio-settings--view'
+        );
+        if (audioSettings) {
+          audioSettings[0].scrollIntoView(
+            { behavior: 'smooth', block: 'start' },
+            150
+          );
+        }
+      });
+    },
```

Comment:
Instead of this can we just add `id` to all the blocks in the profile settings? That way a shareable link can be created for any section of the settings page. We would avoid this logic too with that. 

P.S. We will have to check how will the `$router` respond to this, we might have to make some changes based on this [documentation](https://router.vuejs.org/guide/advanced/scroll-behavior.html)

Reply:
@scmmishra By default, the Vue router (v.3) uses hash mode, but we are using history mode in our configuration. Do I want to change it? Thoughts? 
CC: @pranavrajs @nithindavid

Reply:
We won't have to change the router mode, just a combination of history mode and using a `hash` to route to a part of the page. So the router declaration will be something like this

```js
export const router = new VueRouter({
  mode: "history",
  scrollBehavior(to, from, savedPosition) {
    if (to.hash) {
      return {
        selector: to.hash,
        // , offset: { x: 0, y: 10 }
      };
    }
  },
  routes,
});

```

Reply:
@scmmishra I tried adding it, and it's not working. You can see it in the loom video.
https://www.loom.com/share/e403351c920442148f9f67240ebe8e12

Correct me if anything is wrong.

Reply:
@iamsivin I tried to do this at well, seems like it's not working because the scroll is not on the body element, it's an overflowing inner element instead. Perhaps this can be fixed (Separately of course)

For now the current implementation should be fine, but maybe we can add a TODO there referencing this conversation